### DISCUSSION TITLE
Welcome to CPU Defense discussions!

### DISCUSSION THREAD

**Discussion by ochadenas** [ochadenas/cpudefense#116] [Announcements]
Hi all,

thanks to everyone for your interest in this little game, and for all your feedback and suggestions!

This page is meant to provide room for talk on further development, game play, critics and suggestions. In fact, I'm getting a bit overwhelmed by the amount of issues, and it's becoming difficult to manage; so I'd like to ask you to have discussions rather here, and keep the [issues space](https://github.com/ochadenas/cpudefense/issues) for bug reports and such.

**Comment by Mortis03** [comment]
Love the game!  Up to lvl 56 in endless with all hero's maxed.

**Comment by naim5am** [comment]
Hi. I'm hooked. Somehow it gives me space to think and ruminate on strategies and techniques to manage flow. Thinking in circuits conceptually goes a long way. Thanks. 

I'm on level 240 with I don't know how many coins. I'm stuck with a new issue, *overheating CPU*. I couldn't find anything on overheating in the wiki. Any hints? I tried removing the resistors when I got to high SHR levels and CLK speeds, but the temperature kept rising.

**Reply by naim5am** [reply]
I found that staying at clock speeds lower than 8 would keep the heat managable. Still dunno if choice of reaistors has an effect.

**Reply by Arniiiii** [reply]
you can upgrade a hero that reduces generated heat. I maxed out him and he has -100% reduction at last level. Claude Chappe

https://github.com/ochadenas/cpudefense/wiki/Heroes
